build-docker.sh

```diff
@@ -324,6 +324,7 @@ for BITCOIN_ONLY in ${VARIANTS_legacy[@]}; do
 
   DIRSUFFIX=${BITCOIN_ONLY/1/-bitcoinonly}
   DIRSUFFIX=${DIRSUFFIX/0/}
+  DIRSUFFIX="-T1B1${DIRSUFFIX}"
```

Comment:
What is this?

Reply:
It adds the model name to the legacy build directory name, so it follows the same pattern as core. Makes the script that prints the fingerprints simpler.

Reply:
Ah yes, nevermind. I thought this was added unconditionally for all models. But now I see we have two build loops. One for core and one for legacy. 👍